CarePartner/Models/FolloweeManager.swift

```diff
@@ -105,6 +105,26 @@ class FolloweeManager: ObservableObject {
         followees[followee.userDetails.id] = followee
         followee.delegate = self
     }
+    
+    func acceptInvite(pendingInvite: PendingInvite) async -> Bool {
+        do {
+            try await tidepoolClient.api.acceptInvite(invitedByUserId: pendingInvite.userDetails.id, key: pendingInvite.key)
+            return true
+        } catch {
+            print("Could not accept invite from \(pendingInvite.userDetails.fullName): \(error)")
```

Comment:
Just a question here on the printing instead of logging when this eventually goes to prod?

Reply:
good point. I was following the existing pattern, but I'll update to at least log it